pulsar-broker/pom.xml

```diff
@@ -268,6 +268,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
```

Comment:
you need to remove the aws s3 dependency too.

Reply:
oh, Seems as you commented, the  AWSCredentials related things could not get out.

Reply:
We shouldn't need the full s3 dependency now, just whatever AWSCredentials is in.